src/components/HeroCopyElement/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState, useEffect } from "react";
+import Typography from "root/components/Typography";
+
+import "./index.css";
+
+function HeroCopyElement() {
+  const [seconds, setSeconds] = useState(0);
+  const [position, setPosition] = useState(0);
+  const [isActive, setIsActive] = useState(true);
+  const textPlaceholder = ["For founders", "For companies", "For everyone"];
+
+  function getClassName(place) {
+    if (place === position) {
+      return "line-klein-blue";
+    }
+
+    return "line-baby-blue";
+  }
+
+  function reset(value) {
+    setSeconds(value);
+    setPosition(value);
+    setIsActive(false);
+    setInterval(() => {
+      setIsActive(true);
+    }, 5000);
+  }
+
+  useEffect(() => {
+    let interval = null;
+
+    if (isActive) {
+      interval = setInterval(() => {
+        setSeconds(secs => secs + 1);
+        setPosition((seconds + 1) % 3);
+      }, 5000);
+    } else if (!isActive && seconds !== 0) {
+      clearInterval(interval);
+    }
+
+    return () => clearInterval(interval);
+  }, [isActive, seconds]);
+
+  return (
+    <div styleName="copy-section">
+      <div styleName="lines">
+        <button
+          type="button"
+          aria-label="clickable info 1"
+          styleName={getClassName(0)}
+          style={{ "--top": "0" }}
+          onClick={() => reset(0)}
+        />
+        <button
+          type="button"
+          aria-label="clickable info 2"
+          styleName={getClassName(1)}
+          style={{ "--top": "1" }}
+          onClick={() => reset(1)}
+        />
+        <button
+          type="button"
+          aria-label="clickable info 3"
+          styleName={getClassName(2)}
+          style={{ "--top": "2" }}
+          onClick={() => reset(2)}
+        />
+      </div>
+      <div styleName="message">
+        <Typography color="oxford-blue">{textPlaceholder[position]}</Typography>
+        <Typography color="klein-blue" weight="bold" variant="h2">
+          We materialize ideas
```

Comment:
This text also changes

Reply:
Oops! Did not see that... It's a quick update, I'll wait for this deploy to finish so we can discuss animations, and then add these changes also.